packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -363,7 +368,7 @@ module.exports = {
     }
 
     // Update the user.
-    await strapi.query('user', 'users-permissions').update({ id: user.id }, user);
+    await strapi.query('user', 'users-permissions').update({ id: user.id }, { resetPasswordToken });
```

Comment:
you can remove the user.resetPasswordToken : resetPasswordToken if you do that :)

Reply:
Same as above, I wasn't sure the property set on the `user` wasn't used line 344. (Not familiar with the lodash / the syntax). Fixing them now.